### DISCUSSION TITLE
FSS over time

### DISCUSSION THREAD

**Discussion by CPKalb** [dtcenter/METplus#1625] [Configuration] [answered]
Hi:

Is there a way to aggregate FSS over time?  I see that it's part of the NBRCNT line output from grid_stat.  However, in looking at the MET users guide for stat_analysis (Table 15.5), I didn't see a way to aggregate this line over time.

Thanks,
Tina

**Comment by JohnHalleyGotway** [comment, marked as answer]
@CPKalb the FSS column exists in the NBRCNT line type. And [this section](https://met.readthedocs.io/en/latest/Users_Guide/stat-analysis.html#aggregated-values-from-multiple-stat-lines) of the User's Guide includes NBRCNT in the list of STAT line types which can be aggregated by running the ```-job aggregate``` option.

I see that the Stat-Analysis chapter includes this [nice table](https://met.readthedocs.io/en/latest/Users_Guide/stat-analysis.html#id9) summarizing the input/output line type combinations supported by ```-job aggregate_stat```. Do you think having a similar table in [this section](https://met.readthedocs.io/en/latest/Users_Guide/stat-analysis.html#job-aggregate) to clearly define which line types can be aggregated would have made the functionality more clear to you?

**Comment by CPKalb** [comment]
Ah...  What happened is that I'm used to the CNT line using aggregate_stat from a SL1L2 line type, so was looking for something similar with the NBRCNT line.  Therefore, I didn't pay attention to the aggregate section (which is where I needed to look).  I think it would be helpful to have a listing in section 15.3.3.3, so it's in a similar area to where the aggregate_stat listing is located.

**Reply by JohnHalleyGotway** [reply]
I took the easy route and just copied/pasted some content from earlier in the Stat-Analysis chapter to this section:
https://met.readthedocs.io/en/develop/Users_Guide/stat-analysis.html#job-aggregate

Hopefully that will suffice.

Please go ahead and "Mark as answer" one of the responses if you're all set.
